libs/ui/packages/native/src/components/Layout/List/VerticalTimeline/TimelineIndicator.tsx

```diff
@@ -60,23 +114,36 @@ export type Props = FlexProps & {
   status: ItemStatus;
   isFirstItem?: boolean;
   isLastItem?: boolean;
+  topHeight?: number;
 };
 
-export default function TimelineIndicator({ status, isFirstItem, isLastItem, ...props }: Props) {
+export default function TimelineIndicator({
+  status,
+  isFirstItem,
+  isLastItem,
+  topHeight,
+  ...props
+}: Props) {
   const { colors } = useTheme();
 
   return (
     <Flex flexDirection="column" alignItems="center" {...props}>
-      <TopSegment status={status} hidden={isFirstItem} />
-      <CenterSegment status={status} isLastItem={isLastItem}>
+      <TopSegmentSvg
+        status={status}
+        hidden={isFirstItem}
+        height={PixelRatio.roundToNearestPixel(topHeight || topSegmentDefaultHeight)}
+      />
+      <CenterCircle status={status} isLastItem={isLastItem}>
         {status === "completed" && (
           <CircledCheckSolidMedium
             color={isLastItem ? colors.success.c100 : colors.primary.c80}
-            size={24}
+            size={20}
           />
         )}
-      </CenterSegment>
-      <BottomSegment status={status} hidden={isLastItem} />
+      </CenterCircle>
```

Comment:
Can this be a "self closing" component: `<CenterCircle />` here ?

Reply:
It could be if we "absolute" position the `CircledCheckSolidMedium`, but for centering purposes it's easier like that.

Reply:
I'm dumb, never mind i read it too fast, it's good like this !